autoscheduler/frontend/src/tests/ui/CourseSelectCard.test.tsx

```diff
@@ -860,12 +970,66 @@ describe('Course Select Card UI', () => {
         }, '201931'));
 
         // assert
-        expect(getByText('ONLINE')).toBeTruthy();
+        expect(getByText('N/A')).toBeTruthy();
+      });
+
+      test('for 00:00 meeting times & hybrid section', () => {
+        // arrange
+        const store = createStore(autoSchedulerReducer, applyMiddleware(thunk));
+
+        const {
+          getByText,
+        } = render(
+          <Provider store={store}><SectionSelect id={0} /></Provider>,
+        );
+
+        const testSection = new Section({
+          id: 0,
+          crn: 0,
+          subject: 'CSCE',
+          courseNum: '121',
+          sectionNum: '200',
+          minCredits: 3,
+          maxCredits: null,
+          currentEnrollment: 0,
+          maxEnrollment: 0,
+          instructor: new Instructor({
+            name: 'Test',
+          }),
+          honors: false,
+          web: false,
+          grades: null,
+        });
+        const testMeeting = new Meeting({
+          id: 1,
+          building: '',
+          meetingDays: new Array(7).fill(true),
+          startTimeHours: 0,
+          startTimeMinutes: 0,
+          endTimeHours: 0,
+          endTimeMinutes: 0,
+          meetingType: MeetingType.LEC,
+          section: testSection,
+        });
+
+        const sectionSelected = {
+          section: testSection,
+          meetings: [testMeeting],
+          selected: false,
+        };
+
+        // Add the SectionSelected type to the store so it shows up in the SectionSelect component
+        store.dispatch<any>(updateCourseCard(0, {
+          sections: [sectionSelected],
+        }, '201931'));
+
+        // assert
+        expect(getByText('N/A')).toBeTruthy();
       });
     });
 
     describe('does not show ONLINE', () => {
-      test('for 00:00 meeting times that are not online sections', () => {
```

Comment:
@ryan-conn What was the use case for this test again? I think it was some kind of research/study abroad section or something?

Reply:
Yeah it was for a research section (although there are probably others), but since we're never showing ONLINE as the time anymore it's fine that you removed this test